imports/plugins/core/layout/client/components/coreLayout.js

```diff
@@ -1,21 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 import classnames from "classnames";
-import { Components, registerComponent } from "@reactioncommerce/reaction-components";
+import { getComponent, registerComponent } from "@reactioncommerce/reaction-components";
 import Blaze from "meteor/gadicc:blaze-react-component";
 import { Template } from "meteor/templating";
 
 const CoreLayout = ({ actionViewIsOpen, structure }) => {
-  const { layoutFooter, template } = structure || {};
+  const { layoutHeader, layoutFooter, template } = structure || {};
 
   const pageClassName = classnames({
     "page": true,
     "show-settings": actionViewIsOpen
   });
 
+  const headerComponent = layoutHeader && getComponent(layoutHeader);
+  const footerComponent = layoutFooter && getComponent(layoutFooter);
+
   return (
     <div className={pageClassName} id="reactionAppContainer">
-      <Components.NavBar />
+
+      { headerComponent && React.createElement(headerComponent, {}) }
```

Comment:
No extra space with the `{}` in JSX templates.

Reply:
Done. Btw, is this enforced by eslint?